server.rb

```diff
@@ -10,9 +10,12 @@
 NEW_MESSAGE_CHANNEL = 'new_message_channel'.freeze
 UPDATED_CLIENT_CHANNEL = 'updated_client_channel'.freeze
 MESSAGE_LIST_KEY = 'message_list'.freeze
+CLIENT_IDS_KEY = 'client_ids'.freeze
 
 # Clear out any old messages when we boot up
 Redis.new(url: ENV['REDIS_URL']).del(MESSAGE_LIST_KEY)
+Redis.new(url: ENV['REDIS_URL']).del(CLIENT_IDS_KEY)
```

Comment:
Might as well reuse the connection here:
redis = Redis.new(url: ENV['REDIS_URL'])
redis.del(MESSAGE_LIST_KEY)
redis.del(CLIENT_IDS_KEY)